compiler/src/dmd/hdrgen.d

```diff
@@ -2000,8 +2004,31 @@ private void visitVarDecl(VarDeclaration v, bool anywritten, ref OutBuffer buf,
         buf.writestring(" = ");
         vinit(v);
     }
+    if (commentIt)
+        buf.writestring("+/");
```

Comment:
Looking at the output in `compiler/test/compilable/ctod.i`, it looks like this function is not the right place to emit the comment start and end tokens, since the `;` is emitted by the callers. I suggest you see if you can update the callers instead:
* `visitCompoundDeclaration`
* `visitVarDeclaration`
* `visitDeclaration`

Reply:
This is the simplest solution, and the extra ; is harmless.

Reply:
The proposed change would put the `/+ +/` in three places rather than one.